tests/test_process.py

```diff
@@ -108,6 +111,157 @@ def test_load_rain_file(rain_data_file):
     ]
 
 
+def test_load_rain_file_flanders(rain_data_file_flanders):
+
+    """Valid rainfall data should be parsed to rain DataFrame"""
+    rainfall_data = load_rain_file(rain_data_file_flanders, load_rain_file_flanders)
+
+    assert isinstance(rainfall_data, pd.DataFrame)
+
+    assert list(rainfall_data.columns) == [
+        "datetime",
+        "station",
+        "rain_mm",
+        "year",
+        "tag",
+    ]
+
+    assert list(rainfall_data["station"].unique()) == ["FL_Test"]
+    assert list(rainfall_data.iloc[0].values) == [
+        pd.to_datetime("2024-01-01 00:30:00"),
+        "FL_Test",
+        10.5,
+        2024,
+        "FL_Test_2024",
+    ]
+
+    assert list(rainfall_data.iloc[-1].values) == [
```

Comment:
here you only need to test the specific value, i.e. not all columns, but only the 1.0 value. Same for similar asserts